tests/decorators/decorators.go

```diff
@@ -22,4 +22,6 @@ var (
 	VGPU                 = []interface{}{Label("VGPU")}
 	SRIOV                = []interface{}{Label("SRIOV")}
 	NonRoot              = []interface{}{Label("verify-non-root")}
+	Native               = []interface{}{Label("native")}
```

Comment:
Can we add `-ssh` to have a little bit more context?

Reply:
Done!